_data/insights-videos.yaml

```diff
@@ -5,6 +5,10 @@ futurevideos:
 
 pastheadline: "Archived Podcast Videos"
 
+- title: "EPISODE #80: Neo4j with Quarkus"
+  link: 
+  date: "February 14, 2022"
+  authors: "Michael Simons"
```

Comment:
This block is not in the right place. It should be moved just below `futurevideos:`.